docs/requirements.in

```diff
@@ -4,4 +4,4 @@
 # When updating dependencies here:
 # - Install pip-tools with `pip install pip-tools`
 # - Use `pip-compile requirements.in` to update transitive dependency versions in requirements.txt
```

Comment:
Did you run this to update the transitive dependencies?

Reply:
Fixed!